README.md

```diff
@@ -1 +1,131 @@
-# azure-functions-openai-aisearch-python
\ No newline at end of file
+<!--
+---
+name: Azure Function using OpenAI trigger and bindings extension to highlight OpenAI retrieval augmented generation with Azure AI Search.
+description: This repository contains an Azure Function using OpenAI trigger and bindings extension to highlight OpenAI retrieval augmented generation with Azure AI Search. The sample uses managed identity.
+page_type: sample
+products:
+- azure-functions
+- azure
+- entra-id
+urlFragment: azure-functions-openai-aisearch-dotnet8
```

Comment:
```suggestion
urlFragment: azure-functions-openai-aisearch-python
```

Need to pythonize all the metadata like this

Reply:
fixed